### PR TITLE
IRSA-438:Make IRSA Viewer default overlay color be more visible

### PR DESCRIPTION
**Pull request by lznakano**
Changed the default  overlay color to red

### CODE REVIEW

File: src/firefly/js/drawingLayers/ActiveTarget.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ var idCnt=0;
 
 
 function creator(initPayload, presetDefaults) {
-    var drawingDef= makeDrawingDef('blue');
+    var drawingDef= makeDrawingDef('red');
```
Comment (robyww) [inline src/firefly/js/drawingLayers/ActiveTarget.js]: Do we want to do `red`.  We use `red` for a lot of layers and we already take heat about using it.

Comment (ejoliet) [inline src/firefly/js/drawingLayers/ActiveTarget.js]: Please, change to pink or magenta. Thanks.

**Comment by robyww** [comment]
It simple to make the line thicker as well. I could talk to Lijun about how tomorrow.

File: src/firefly/js/visualize/draw/Drawer.js [hunk 2]
```diff
@@ -624,7 +624,7 @@ function drawConnector(ctx, def, csysConv, dc, obj, lastObj) {
 
 function drawChunkOptimized(drawList, params, ctx) {
     if (!drawList.length) return;
-    DrawUtil.beginPath(ctx,params.drawingDef.color,1);
+    DrawUtil.beginPath(ctx,params.drawingDef.color,params.drawingDef.lineWidth);
```
Comment (robyww) [inline src/firefly/js/visualize/draw/Drawer.js]: Good catch, I would have missed this.

File: src/firefly/js/drawingLayers/ActiveTarget.js [hunk 3]
```diff
@@ -27,11 +27,14 @@ var idCnt=0;
 
 
 function creator(initPayload, presetDefaults) {
-    var drawingDef= makeDrawingDef('magenta');
+    var drawingDef= makeDrawingDef('magenta',4 );
```
Comment (robyww) [inline src/firefly/js/drawingLayers/ActiveTarget.js]: the `4` is unnecessary. It takes an object, for example:
`var drawingDef= makeDrawingDef('magenta', 
{lineWidth:4, size:6, 
fontWeight:'bolder', 
   symbol: DrawSymbol.CIRCLE } );`

File: src/firefly/js/visualize/draw/DrawingDef.js [hunk 4]
```diff
@@ -88,7 +88,7 @@ export function makeDrawingDef(color= 'red', presetDefaults= {}) {
     const def= {
         color,
         symbol: DrawSymbol.X,
-        lineWidth:1,
+        lineWidt:1,
```
Comment (robyww) [inline src/firefly/js/visualize/draw/DrawingDef.js]: I think you lost an `h`

File: src/firefly/js/visualize/draw/PointDataObj.js [hunk 5]
```diff
@@ -163,6 +163,7 @@ function makeDrawParams(pointDataObj,def) {
     const fontWeight= pointDataObj.fontWeight || def.fontWeight || 'normal';
     const fontStyle= pointDataObj.fontStyle || def.fontStyle || 'normal';
     const textLoc= pointDataObj.textLoc || def.textLoc || TextLocation.DEFAULT;
+    const lineWidth =pointDataObj.lineWidth || def.lineWidth || '1';
```
Comment (robyww) [inline src/firefly/js/visualize/draw/PointDataObj.js]: `'1'` should not be a string

**Comment by robyww** [comment]
There are several little things that should be fixed. Make sure you test it with catalogs to make sure all is working since catalogs use `PointDataObj`.  Otherwise look good.

**Comment by lznakano** [comment]
Thank Trey very much for catching those issues.  I fixed them  and tested catalog search.  It seems all fine.  As Emmanuel noted, this ticket only asked for the color change, thus, I did not increase the lineWidth.  But the codes are fixed now.  The lineWidth can be changed later.  I will add comments in the ticket so that the information can be recorded.   Can you both have another look please?

**Pull request merged**